skimage/transform/_thin_plate_splines.py

```diff
@@ -0,0 +1,306 @@
+import numpy as np
+import scipy as sp
+
+
+class TpsTransform:
```

Comment:
A docstring for the class here is still missing. I think you can move most of the docstring from `__call__` here. `__call__` isn't rendered in the [HTML documentation](https://output.circle-artifacts.com/output/job/56bdac54-7c2b-425b-909d-e54f7a7bac09/artifacts/0/doc/build/html/api/skimage.transform.html#skimage.transform.TpsTransform).